src/main/java/com/mark0wka/controllers/CinemaController.java

```diff
@@ -18,74 +18,138 @@
+    }
+
+    @GetMapping("/halls/{cinemaName}")
+    public Set<EntityObject> getHallsByCinemaName(@PathVariable String cinemaName) {
+        Set<EntityObject> result = new HashSet<>();
+        List<Attribute> id = attributeRepository.findByAttrIdAndValue(2, String.valueOf(objectRepository.findEntityObjectByName(cinemaName).getObjectId()));
+        id.forEach(x -> result.add(objectRepository.findEntityObjectByObjectId(x.getObjectId())));
+        return result;
+    }
+
+    @GetMapping("/sessions/{filmId}")
+    public Set<EntityObject> getSessionsByFilmId(@PathVariable int filmId) {
+        Set<EntityObject> result = new HashSet<>();
+        List<Attribute> id = attributeRepository.findByAttrIdAndValue(8, String.valueOf(filmId));
+        id.forEach(x -> result.add(objectRepository.findEntityObjectByObjectId(x.getObjectId())));
+        return result;
+    }
+
+    @GetMapping("/seats/{sessionId}")
+    public Set<EntityObject> getSeatsBySessionId(@PathVariable int sessionId) {
+        Set<EntityObject> result = new HashSet<>();
+        List<Attribute> id = attributeRepository.findByAttrIdAndValue(22, String.valueOf(sessionId));
+        id.forEach(x -> result.add(objectRepository.findEntityObjectByObjectId(x.getObjectId())));
+        return result;
+    }
+
+    @GetMapping("/hello")
+    public String hello() {
+        return "hello!";
     }
 
 
     @PostMapping("/create/cinema")
     public EntityObject createCinema(@RequestBody Request request) {
-        if (!new CreateServices().cinemaValidator(request.getName(), ObjRepository)){
+        if (!new CreateServices().cinemaValidator(request.getName(), objectRepository)){
             System.out.println("Already exists");
             return null;
         }
         return saveData(request);
     }
 
-
     @PostMapping("/create/film")
     public EntityObject createFilm(@RequestBody Request request) {
         return saveData(request);
     }
 
     @PostMapping("/create/hall")
-    public Set<EntityObject> createHall(@RequestBody Request request) {
+    public EntityObject createHall(@RequestBody Request request) {
+        return saveData(request);
+    }
+
+    @PutMapping("/update/seat")
+    public List<EntityObject> updateSeat(@RequestBody ReservedSeatReq request) {
+        List<Integer> seatsId = request.getSeatsId();
+        boolean isAnyConflict = seatsId.stream()
+                .map(seat -> attributeRepository.findAttributeByObjectIdAndAttrId(seat, 23))
+                .map(Attribute::getValue)
+                .anyMatch("true"::equals);
+        if (isAnyConflict) {
+            //throw exceptiom
+        }
+        return seatsId.stream()
+                .map(objectRepository::getById)
+                .map(seat -> bookSeat(seat, request.getUser()))
+                .collect(Collectors.toList());
+    }
+
+    private EntityObject bookSeat(final EntityObject seat, final int userId) {
+        seat.setValue(23, "true");
```

Comment:
Please create SQLConstants class and transfer all of the IDs to it
Its not readable